api/dao/hierarchy.py

```diff
@@ -572,7 +572,8 @@ def _update_container_nulls(base_query, update, container_type):
     return config.db[coll_name].find_one(base_query)
 
 
-def merge_fileinfos(parsed_files, infos):
+# NOTE skip coverage since this function is currently not used
+def merge_fileinfos(parsed_files, infos): # pragma: no cover
```

Comment:
Thanks for calling that out - I'm doing some work in this file and will just remove it in that PR.